packages/gamut/src/ProgressBar/index.tsx

```diff
@@ -23,52 +23,115 @@ export type ProgressBarProps = {
   percent: number;
 
   /**
-   * Style overrides to apply on top of the theme, if any.
+   * Base variant display themes.
    */
-  style?: ProgressBarStyle;
-
-  /**
-   * Base color theme to extend from.
-   */
-  theme: 'blue' | 'yellow';
+  variant: 'blue' | 'yellow' | 'borderedDarkMode' | 'borderedLightMode';
```

Comment:
Nit: "mode" is redundant, and IIRC we typically dash-lower-case these? Definitely not a blocker if you don't want it to be. Though eventually we should probably standardize.

```suggestion
  variant: 'blue' | 'yellow' | 'bordered-dark' | 'bordered-light';
```

Reply:
i found that `variant` doesn't love dash-lower-case, so i've changed these to `borderedLight` + `borderedDark`

Reply:
Weird, what exactly happens when you try dashcase?

IIRC this should work:
```tsx
variant({
  ['bordered-dark']: {}
})
```
But who knows if that causes the whole thing to explode.

Reply:
i wasn't wrapping them in `[]`, i'll try it that way!